src/baseInterceptor.js

```diff
@@ -23,101 +23,144 @@ function baseInterceptor() {
+          'color': 'black with ' + trans + '% tranparency',
           rgb
         });
       }
-    } else if (arguments[0] > 240) {
-      const rgb = `(255, 255, 255)`;
+    } else if (colArgs[0] > 240) {
+      const rgb = '(255, 255, 255)';
       if (trans === 0) {
         return ({
-          'color': `white`,
+          'color': 'white',
           rgb
         });
       } else {
         return ({
-          'color': `white with ` + trans + `% tranparency`,
+          'color': 'white with ' + trans + '% tranparency',
           rgb
         });
       }
     } else {
-      const rgb = `(` + Math.round(arguments[0]) + `, ` + Math.round(arguments[0]) + `, ` + Math.round(arguments[0]) + `)`;
+      const rgb = '(' + Math.round(colArgs[0]) + ', ' + Math.round(colArgs[0]) + ', ' + Math.round(colArgs[0]) + ')';
       if (trans === 0) {
         return ({
-          'color': `gray`,
+          'color': 'gray',
           rgb
         });
       } else {
         return ({
-          'color': `gray with ` + trans + `% tranparency`,
+          'color': 'gray with ' + trans + '% tranparency',
           rgb
         });
       }
     }
-  } else if (arguments.length === 1) {
-    if (!(typeof(arguments[0])).localeCompare(`number`)) {
+  } else if (colArgs.length === 1) {
+    if (!(typeof(colArgs[0])).localeCompare('number')) {
       // assuming that we are doing RGB - this would be a grayscale number
-      if (arguments[0] < 10) {
-        const rgb = `(0, 0, 0)`;
+      if (colArgs[0] < 10) {
+        const rgb = '(0, 0, 0)';
         return ({
-          'color': `black`,
+          'color': 'black',
           rgb
         });
-      } else if (arguments[0] > 240) {
-        const rgb = `(255, 255, 255)`;
+      } else if (colArgs[0] > 240) {
+        const rgb = '(255, 255, 255)';
         return ({
-          'color': `white`,
+          'color': 'white',
           rgb
         });
       } else {
-        const rgb = `(` + arguments[0] + `, ` + arguments[0] + `, ` + arguments[0] + `)`;
+        const rgb = '(' + colArgs[0] + ', ' + colArgs[0] + ', ' + colArgs[0] + ')';
         return ({
-          'color': `grey`,
+          'color': 'grey',
           rgb
         });
       }
-    } else if (!(typeof(arguments[0])).localeCompare(`string`)) {
-      if (!arguments[0].charAt(0).localeCompare(`#`)) {
-        return (getHexname(arguments));
-      } else if (arguments[0].match(/rgba/)) {
-        return (RGBAString(arguments));
-      } else if (arguments[0].match(/rgb/)) {
-        return (RGBString(arguments));
+    } else if (!(typeof(colArgs[0])).localeCompare('string')) {
+      if (!colArgs[0].charAt(0).localeCompare('#')) {
+        return (getHexname(colArgs));
+      } else if (colArgs[0].match(/rgba/)) {
+        return (RGBAString(colArgs));
+      } else if (colArgs[0].match(/rgb/)) {
+        return (RGBString(colArgs));
+      } else if(htmlColors.findIndex(el => el.name === colArgs[0].toUpperCase())){
```

Comment:
@lm-n if use enters an invalid string, like 'REDD', I believe it defaults to white. Can we ensure the output also says `white`? Right now it breaks the entire sketch

Reply:
I think it might also make sense to be explicit that it defaulted to white. Thoughts?

Reply:
Yup will make sure it goes to white. @MathuraMG what do you think of Mithru's idea?